api/search/views.py

```diff
@@ -668,6 +668,8 @@ def search_fields(self):
             'provider',
             'title',
             'tags',
+            'school_type_choices',
```

Comment:
I think these would be `schoolType` and `studyDesign`, but I'm also not sure if they're used.

Reply:
apparently they are not used, I tested searching for them via string `?q=test school type` as school_type was `test school type` and that worked. So I'm not sure what to think.

Reply:
I noticed these fields are used in the `SearchParser` parser, so I'm putting the attributes in the list in case they are used in some complex filtering or something.